src/server/index.ts

```diff
@@ -92,6 +94,41 @@ router.post<
   });
 });
 
+
+router.post("/internal/post-create", async (_req, res): Promise<void> => {
+  const { subredditName } = context;
+  if (!subredditName) {
+    res.status(400).json({
+      status: "error",
+      message: "subredditName is required",
+    });
+    return;
+  }
+  reddit.submitPost({
+    runAs: 'USER',
```

Comment:
why user?

Reply:
I think this should be a default inside of our platform if we want it as a default, not the template. I'm also not sure if things like text fallback can be updated if it's posted as user. I know that was an issue last time I tried it.